tests/baselines/reference/classMemberWithMissingIdentifier2.errors.txt

```diff
@@ -14,6 +14,7 @@ tests/cases/compiler/classMemberWithMissingIdentifier2.ts(3,1): error TS1128: De
 !!! error TS1146: Declaration expected.
                ~
 !!! error TS1005: ';' expected.
+!!! related TS1007 tests/cases/compiler/classMemberWithMissingIdentifier2.ts:1:9: The parser expected to find a '}' to match the '{' token here.

```

Comment:
So it looks like the heuristic on this is actually...not working correctly given

https://github.com/microsoft/TypeScript/blob/7f5994958b33dbbaaeccd4b6eef8584c960d4d71/src/compiler/parser.ts#L1136